confirmer/confirmer.go

```diff
@@ -121,7 +121,7 @@ func (confirmer *Confirmer) confirm(ctx context.Context, transaction tx.Tx) {
 		case <-ctx.Done():
 			return
 		case response := <-req.Responder:
-			if response.Error == nil || strings.Contains(response.Error.Message, "current status = done") {
+			if response.Error == nil || strings.Contains(response.Error.Message, "status=done") {
```

Comment:
Will this still work for the old error message since it contained "status = done"?

Reply:
Good point, should probably check both